mmv1/products/discoveryengine/ChatEngine.yaml

```diff
@@ -125,7 +129,12 @@ properties:
         type: NestedObject
         description: |
           The configuration to generate the Dialogflow agent that is associated to this Engine.
-        required: true
+          Exactly one of `agent_creation_config` or `dialogflow_agent_to_link` must be set.
+        immutable: true
+        ignore_read: true
```

Comment:
why are we adding `ignore_read` to this one here? Did it not correctly read before?

Reply:
ignore_read is set on the parent field here https://github.com/GoogleCloudPlatform/magic-modules/blob/bbf9f4086880429a90c5272e117d24541698ac68/mmv1/products/discoveryengine/ChatEngine.yaml#L126
So I decided to repeat it for both nested fields to have it set more explicit in the code, but you are right, it can be removed since parent field has it set already

Reply:
thanks for the context